src/views/FindShipment.vue

```diff
@@ -0,0 +1,241 @@
+            <ion-list-header>{{ $t("Date") }}</ion-list-header>
+            <ion-item>
+              <ion-label>{{ $t("Arrival date") }}</ion-label>
+              <ion-chip>
+                <ion-icon :icon="calendarOutline" />
+                <ion-input type="date" />
+                <ion-icon :icon="closeCircle" />
+              </ion-chip>
+            </ion-item>
+          </ion-list>
+        </aside>
+
+        <main>
+          <section class="sort">
+            <ion-item lines="none">
+              <ion-icon :icon="documentTextOutline" slot="start" />
+              <ion-label>{{ $t("Show shipment items") }}</ion-label>
+              <ion-toggle slot="end" checked></ion-toggle>
+            </ion-item>
+
+            <ion-item lines="none">
+              <ion-icon :icon="swapVerticalOutline" slot="start" />
+              <ion-label>{{ $t("Sort by") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">Arrival date</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </section>
+
+          <hr />
+
+          <!-- Product section -->
+          <div @click="() => router.push('/shipment')">
+            <section class="section-header">
+              <div class="primary-info">
+                <ion-item lines="none">
+                  <ion-label>
+                    <p>Product store</p>
+                    Shipment external ID
+                    <p>Created date</p>
+                  </ion-label>
+                </ion-item>
+              </div>
+
+              <div class="tags desktop-only">
+                <ion-chip outline>
+                  <ion-icon :icon="calendarOutline" />
+                  <ion-label>Arrival date</ion-label>
+                </ion-chip>
+              </div>
+
+              <div class="metadata">
+                <ion-item lines="none" detail>
+                  <ion-note slot="end">2 {{ $t("items") }}</ion-note>
+                </ion-item>
+              </div>
+            </section>
+
+            <div class="desktop-only">
+              <ion-list>
+                <ion-list-header>
+                  {{ $t("Items") }}
+                  <hr />
+                </ion-list-header>
+                <div v-for="item in 2" :key="item" class="list-item">
+                  <div>
+                    <ion-item lines="none">
+                      <ion-thumbnail slot="start">
+                        <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+                      </ion-thumbnail>
+                      <ion-label>
+                        SKU
+                        <p>{{ $t("Color") }} : color</p>
+                        <p>{{ $t("Size") }}: size</p>
+                      </ion-label>
+                    </ion-item>
+                  </div>
+
+                  <div>
+                    <ion-chip outline>
+                      <!-- TODO -->
```

Comment:
what is the todo?

Reply:
> what is the todo?

We have discussed earlier that some other icon or image we will use here and for now I should use pricetag tag so I have wrote TODO here.

Reply:
Done sir